mailserver.env

```diff
@@ -397,6 +397,8 @@ ENABLE_FETCHMAIL=0
 
 # The interval to fetch mail in seconds
 FETCHMAIL_POLL=300
+# Workaround IMAP bug by splitting fetchmail.cf per poll entry
```

Comment:
Could you provide more context regarding the IMAP bug? This differs from the ENV docs which doesn't mention any IMAP bug to workaround.

Reply:
It's mentioned already in https://docker-mailserver.github.io/docker-mailserver/latest/config/environment/#fetchmail_parallel

See also: https://otremba.net/wiki/Fetchmail_(Debian)#Immediate_Download_via_IMAP_IDLE

Reply:
> It's mentioned already

Ok, but that doesn't describe it as a workaround for an IMAP bug? Nor does the reader of the comment here necessary know the content is any different on the web docs for ENV, so the context matters more.

> See also

Yeah that is much better! 👍 

Ideally our web docs would link to that, and the ENV comment here would just reference that for more context.

```suggestion
# Use multiple fetchmail instances (1 per poll entry in fetchmail.cf)
# Supports multiple IMAP IDLE connections when a server is used across multiple poll entries
# https://otremba.net/wiki/Fetchmail_(Debian)#Immediate_Download_via_IMAP_IDLE
```

That conveys the feature much better with additional context right?